### PR TITLE
More options for RooflinePlotter

### PR DESCRIPTION
**Pull request by FabioLuporini**
better rooflines (multiple rooflines supported!)

### CODE REVIEW

**Comment by FabioLuporini** [comment]
@mlange05 objections ?

**Comment by FabioLuporini** [comment]
@mlange05 Can we merge this?

File: opescibench/plotter.py [hunk 1]
```diff
@@ -315,26 +323,32 @@ def __enter__(self):
...
+            self.ax.loglog(x_roofl, y_roofl, 'k-')
+            self.ax.annotate(**{'xy': (x_roofl[-1] - len(j), y_roofl[idx+1]),
+                                'xytext': (-20, 2), 'textcoords': 'offset points',
+                                'size': 5, 's': j})
 
         # Set axis labelling and generate plot file
-        xlabel='Operational intensity (Flops/Byte)'
-        ylabel='Performance (GFlops/s)'
+        xlabel='Operational intensity (FLOP/Byte)'
+        ylabel='FLOPS'
```
Comment (mlange05) [inline opescibench/plotter.py]: Not a particular fan of the implicit `/sec`, but ok.

**Comment by mlange05** [comment]
Yep, gtg.

**Pull request merged**